components/AppBar/src/ColorThemer/MDCAppBarColorThemer.m

```diff
@@ -23,4 +25,15 @@ + (void)applyColorScheme:(id<MDCColorScheme>)colorScheme
   appBar.headerViewController.headerView.backgroundColor = colorScheme.primaryColor;
 }
 
++ (void)applyExperimentalColorScheme:(MDCExperimentalColorScheme *)colorScheme toAppBar:(MDCAppBar *)appBar {
+  appBar.headerViewController.headerView.backgroundColor = colorScheme.backgroundColor;
```

Comment:
I'm not sure how to make the split between supporting UIAppearance and supporting an instance.
[MDCHeaderView appearanceWhenContainedIn:[MDCHeaderViewControlller]].backgrounColor = colorScheme.backgroundColor;

Reply:
MDCAppBar should expose these as properties in its header.  Since MDCAppBar isn't a UIViewController subclass and doesn't end up in a view hierarchy, we should probably follow the approach taken by TextFields to use  class properties to store default values.